### PR TITLE
JSONFormatBear.py: Fix erroneous message

### PR DESCRIPTION
**Pull request by ZamaSharik**
This fixes display of incorrect message regardless
of enabling json_sort paramater.

Fixes https://github.com/coala/coala-bears/issues/2503

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by abhishalya** [comment]
This is a duplicate of https://github.com/coala/coala-bears/pull/2782

File: .pytest_cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (jayvdb) [inline .pytest_cache/v/cache/lastfailed]: this file shouldnt be in your PR ;-)

File: bears/js/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jayvdb) [inline bears/js/.vscode/settings.json]: this doesnt belong in your PR

**Comment by Sidsharik** [comment]
What is happening with commits? I have resolved the issue. But, still some weird symbols pop-up. I checked using `git log` and there seem to be many commits. Should I squash my commits using the documentation?

**Comment by li-boxuan** [comment]
Yes, please squash your commits. DO NOT push until you have properly squashed.

**Comment by li-boxuan** [comment]
This pull request is messy and going nowhere. Unassigned.

**Pull request closed without merging**